UNLV-CS472-672/2024-S-GROUP5-Munch
# Starting Comment Functionality

## Description / Changes Made

- Linked home page comment button to temporary comment tab
- Comment Page File
- Comment Text Box
- Back Button back to home page
- Send button
- Comment Component File
- Added variables to the what are needed for comments
- added mock data
- started the components for each comment user interface
 

## Screenshots (optional)

![Screenshot_20240326_221718](https://github.com/UNLV-CS472-672/2024-S-GROUP5-Munch/assets/136287986/00858fce-d619-4d3b-878f-4f32dfe19026)


## How to Test

1. Open the app
2. Go to home page
3. click on comment button
4. ensure it goes to the comment page
5. ensure you can type in the box
6. ensure the button is clickable

## Checklist
